### ISSUE TITLE
Account UI resources try to load from admin path instead of frontend path

### ISSUE THREAD

**Issue by BrokenR3C0RD** [keycloak/keycloak#30541]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

account/ui

### Describe the bug

When using Hostname V2 and when `hostname` contains a path different from that of the `hostname-admin` (i.e., `hostname=https://example.com/auth/` and `hostname-admin=https://admin.example.com/admin/`) the account UI (located at `https://example.com/auth/realms/myrealm/account`) tries to load resources from `https://example.com/admin/resources/` instead of `https://example.com/auth/resources/`.

### Version

25.0.0

### Regression

- [ ] The issue is a regression

### Expected behavior

Account console resources should be loaded from `${frontend hostname}/${frontend path}/resources`

### Actual behavior

Resources are loaded from `${frontend hostname}/${admin path}/resources`

### How to Reproduce?

Set up keycloak with `hostname` containing a path that differs from `hostname-admin` (i.e., `hostname=https://example.com/auth/`, `hostname-admin=https://admin.example.com`). Then, try to access the account console while watching requests.

### Anything else?

I'm running my Keycloak instance behind an nginx reverse proxy like so:
```
server {
        listen 80;
        listen 443 ssl;

        server_name example.com;
        location /auth/ {
            proxy_pass          http://10.10.10.2:8080/;
            proxy_set_header    Host                          $host;
            proxy_set_header    X-Real-IP                     $remote_addr;
            proxy_set_header    X-Forwarded-For               $proxy_add_x_forwarded_for;
            proxy_set_header    X-Forwarded-Host              $host;
            proxy_set_header    X-Forwarded-Server            $host;
            proxy_set_header    X-Forwarded-Port              $server_port;
            proxy_set_header    X-Forwarded-Proto             $scheme;
        }
}
server {
        listen 80;
        listen 443 ssl;

        location / {
            proxy_pass          http://10.10.10.2:8080/;
            proxy_set_header    Host                          $host;
            proxy_set_header    X-Real-IP                     $remote_addr;
            proxy_set_header    X-Forwarded-For               $proxy_add_x_forwarded_for;
            proxy_set_header    X-Forwarded-Host              $host;
            proxy_set_header    X-Forwarded-Server            $host;
            proxy_set_header    X-Forwarded-Port              $server_port;
            proxy_set_header    X-Forwarded-Proto             $scheme;
        }
}
```

`/auth/realms/myrealm/login` uses the correct path:
```

        {
            "imports": {
                "rfc4648": "/auth/resources/bpv49/common/keycloak/node_modules/rfc4648/lib/rfc4648.js"
            }
        }
```

This can be mitigated in the meantime by adding a `proxy_pass` directive for `/resources` on `example.com`, though this is less than ideal:

```
        location /resources/ {
            proxy_pass          http://10.10.10.2:8080/resources/;
            proxy_set_header    Host                          $host;  
            proxy_set_header    X-Real-IP                     $remote_addr;
            proxy_set_header    X-Forwarded-For               $proxy_add_x_forwarded_for;
            proxy_set_header    X-Forwarded-Host              $host;
            proxy_set_header    X-Forwarded-Server            $host;  
            proxy_set_header    X-Forwarded-Port              $server_port;
            proxy_set_header    X-Forwarded-Proto             $scheme;
        }
```

**Comment by BrokenR3C0RD** [comment]
Updated as I realized the error actually stems from `hostname-admin`'s path being used instead of `hostname`: https://github.com/keycloak/keycloak/blob/e89b4182b64802c777b435513db9138ee52e05ec/services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java#L105

**Comment by jonkoops** [comment]
This sound like a duplicate of #30432. @BrokenR3C0RD can you confirm this issue still exists in 25.0.1?

**Comment by BrokenR3C0RD** [comment]
This still repros in Keycloak 25.0.1; the fix may have changed the behavior back to the original titling of this issue now.

If I go to `https://auth.example.com/auth/realms/myrealm` in my example (not publically accessible), the page loads the resources correctly from `/auth/resources/`. If I go to the admin console at `https://admin.auth.example.com/admin/admin/master/console` (the extra `/admin` being added through `hostname-admin=https://admin.auth.example.com/admin/`), it loads its resources from `/admin/resources/`. However, if I go to `https://auth.example.com/auth/realms/myrealm/account`, it tries to load its resources from `/resources` instead of `/auth/resources` (which causes the page to get stuck at `Loading the Account Console`).

**Comment by BrokenR3C0RD** [comment]
Here is an example `keycloak.conf` and nginx vhost configuration for reproduction

**Comment by jonkoops** [comment]
Thanks for confirming and providing some more steps to reproduce @BrokenR3C0RD, I'll have a look.

**Comment by jonkoops** [comment]
@BrokenR3C0RD have you tested this with the legacy hostname as well? Does that still work?

**Comment by jonkoops** [comment]
With some help from @vmuzikar I was able to reproduce this. The issue is that the `resourceCommonUrl` injected into the template for the account console is set to the the admin hostname. This bug has been in there since this [functionality was introduced](https://github.com/keycloak/keycloak/commit/35931d60ebe15f6cdc995b31cac7be66d2f1bef5#diff-7fab7d2b4302bc00a2ba67bd7eee3dea5fdae47265031251cf8873d4aedaa024R93) over 4 years ago, but likely went largely unnoticed until we recently started relying on it more in the updated account console.

For these reasons I will consider this a regression, which will need a backport for Keycloak 25 and RHBK 22 & 24.
